lib/modules/filteReddit/Filter.js

```diff
@@ -56,7 +55,11 @@ export default class Filter<Parsed: { [key: any]: any }> {
 	}
 
 	createElement() {
-		this.element = $(filterlineFilterTemplate(this))[0];
+		this.element = $(`
```

Comment:
Maybe we should have a `module.utils.html(markup: string)` that abstracts away `$(markup)[0]`? We could probably even skip jQuery (unless it adds security) with something like `x = document.createDocumentFragment(); x.innerHTML = markup; return x`